dbms/src/Flash/Coprocessor/StreamingDAGResponseWriter.cpp

```diff
@@ -78,37 +58,21 @@ StreamingDAGResponseWriter<StreamWriterPtr, enable_fine_grained_shuffle>::Stream
     }
 }
 
-template <class StreamWriterPtr, bool enable_fine_grained_shuffle>
-void StreamingDAGResponseWriter<StreamWriterPtr, enable_fine_grained_shuffle>::finishWrite()
+template <class StreamWriterPtr>
+void StreamingDAGResponseWriter<StreamWriterPtr>::finishWrite()
 {
```

Comment:
Maybe reorder these methods.
Current: finishWrite(); write(); encodeThenWriteBlocks(); batchWrite()
Better: finishWrite(); write(); batchWrite(); encodeThenWriteBlocks()

Reply:
okk

Reply:
function `batchWrite` is removed.